### ISSUE TITLE
Save and Load system does not save the entity collision state

### ISSUE THREAD

**Issue by HierToo** [Dark-Basic-Software-Limited/GameGuruRepo#477]
The save and load system does not save the collision state of the entities which are changed by lua commands. 
So open doors become impassable after save and load.

**Comment by Teabone3** [comment]
This is very good to know. Will have to do some work arounds with my current projects. I noticed this issue with games on Steam made with GG. Now i know why.

**Comment by tarkus1971** [comment]
Yes I have had this also on countless levels.  My main project is on hold until this is fixed. Not just doors but also if I hide, entities and turn off collision, after a load game, entities are visible again or invisible and the collision is on.

**Comment by LeeBamberTGC** [comment]
You would need to modify the save and load scripting to capture any custom LUA values you want to preserve across game saves.  To auto scan and save ALL globals from all scripts would be a crazy amount of data to preserve. Ideas on an elegant solution welcome, perhaps some kind of special group where globals can be placed which ARE saved with game saves.
